test/mongo-milestone-db.js

```diff
@@ -95,6 +95,7 @@ describe('MongoMilestoneDB', () => {
         mongo.collection('m_testcollection').indexInformation((indexError, indexes) => {
           if (indexError) return done(indexError);
           expect(indexes.d_1_v_1).to.be.ok();
+          expect(indexes['m.mtime_1']).to.be.ok();
           done();
         });
       });
```

Comment:
I didn't notice tests for these new methods. Are they exercised by tests in other modules? We should probably check what the coverage is given that there are a number of different arguments that can be specified in different combinations.

Reply:
Which new methods? The index creation? Has this comment drifted a bit?